aiven/client/cli.py

```diff
@@ -1298,6 +1298,141 @@ def service__connector__restart_task(self):
         project_name = self.get_project()
         self.client.restart_kafka_connector_task(project_name, self.args.name, self.args.connector, self.args.task)
 
+    @arg.project
+    @arg.service_name
+    @arg("schema-id", help="ID of a schema")
```

Comment:
`a schema` => `the schema` would sound better though general convention seems to be just `Schema ID`. Having multiple positional arguments is typically discouraged because remembering the order is difficult and using named parameters is preferred.